scan/lib/scan/runner.rb

```diff
@@ -163,10 +166,15 @@ def copy_simulator_logs
     def open_simulator_for_device(device)
       return unless FastlaneCore::Env.truthy?('FASTLANE_EXPLICIT_OPEN_SIMULATOR')
 
-      UI.message("Killing all running simulators")
-      `killall Simulator &> /dev/null`
+      # For backward compatibility force the reset again without regard for the option.
+      force_quit_simulator_processes
 
       FastlaneCore::Simulator.launch(device)
     end
+
+    def force_quit_simulator_processes
+      UI.message("Killing all running simulators")
```

Comment:
Should this change to `other_action.sh({log:false])`  Then there is no log impacts unless debug mode is enabled.